### ISSUE TITLE
download button not present in publication pages

### ISSUE THREAD

**Issue by jedikalimero** [HuiiBuh/InstagramDownloader#65]
This is not exactly a bug but something that could be improved.

You have added a download button to the thumbnails of a gallery, also to the window that pops up when you click on a thumbnail, but this button does not appear when you are seeing the picture (or video, or album) in its own publication page, like this: https://www.instagram.com/p/B5I4sPegcdB/

Sometimes you receive links to a publication like this in Whatsapp, Telegram, etc. and the only way to use your add on in this case is to open the link, then go to the author's gallery, locate the picture in the thumbnails and then download it (or click on the picture if there are more than one picture in the same entry to use the button under the comments to be able to download the second and further pictures).

I think it wouldn't be too difficult to add the button to the publication pages since most of the code needed is the same as the current button under the comments.

**Comment by jedikalimero** [comment]
Ok, this is weird.
I just clicked in the link at the previous post and your button appeared! But then I opened a new tab, copied the link URL and pasted it in the new tab and the button does not appear! 
This is the same as when you click on the link from Whatsapp or Telegram Windows App. The download button does not appear.

Clicked on the link in my previous post:
![Captura de pantalla 2019-11-28 13 18 19](https://user-images.githubusercontent.com/18701653/69809612-fc835280-11e1-11ea-898c-c9d4f976b1db.png)

Pasted the exact same URL in a new tab:
![Captura de pantalla 2019-11-28 13 18 23](https://user-images.githubusercontent.com/18701653/69809626-01480680-11e2-11ea-9f59-a8ab290de037.png)
